share/spice/political_polls/political_polls.js

```diff
@@ -0,0 +1,67 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_political_polls = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.length == 0 || api_result.error) {
+            return Spice.failed('political_polls');
+        }
+        
+        if(api_result.length > 50){
+            api_result = api_result.splice(50);
+        }
+
+        // Render the response
+        DDG.require("moment.js", function() {
+            Spice.add({
+                id: "political_polls",
+
+                // Customize these properties
+                name: "Polls",
+                data: api_result,
+                meta: {
+                    sourceName: "HuffPost Pollster",
+                    sourceUrl: 'http://elections.huffingtonpost.com/pollster',
+                },
+                templates: {
+                    group: 'text',
+                    options: {
+                        detail: false,
+                        item_detail: false,
```

Comment:
`detail` and `item_detail` should be in the `template` object instead of the `options` object:

```javascript
templates: {
    group: 'text',
    detail: false,
    item_detail: false,
    ...
}
```